lib/SyTest/Synapse.pm

```diff
@@ -115,7 +115,8 @@ sub start
    }
 
    if( defined $db_type ) {
-      $self->${\"clear_db_$db_type"}( %db_args );
+      my $clear_meth = "clear_db_${db_type}";
+      $self->${\$clear_meth}( %db_args );
```

Comment:
If you're going to pull it out to a separate variable then it can just be

    $self->$clear_meth( %db_args );